carbon-p2-plugin/src/main/java/org/wso2/maven/p2/feature/generate/FeatureResourceBundle.java

```diff
@@ -0,0 +1,241 @@
+        this.licence = licence;
+    }
+
+    public File getManifest() {
+        return manifest;
+    }
+
+    public void setManifest(File manifest) {
+        this.manifest = manifest;
+    }
+
+    public URL getPropertyFile() {
+        return propertiesFile;
+    }
+
+    public void setPropertyFile(URL propertiesFile) {
+        this.propertiesFile = propertiesFile;
+    }
+
+    public Properties getProperties() {
+        return properties;
+    }
+
+    public void setProperties(Properties properties) {
+        this.properties = properties;
+    }
+
+    public void setBundles(List<Bundle> bundles) {
+        this.bundles = bundles;
+    }
+
+    public List<Bundle> getBundles() {
+        if(this.bundles == null) {
+            return new ArrayList<>();
+        }
+        return this.bundles;
+    }
+    public List<Feature> getImportFeatures() {
+        return importFeatures;
+    }
+
+    public void setImportFeatures(List<Feature> importFeatures) {
+        this.importFeatures = importFeatures;
+    }
+
+    public List<Feature> getIncludeFeatures() {
+        if(includedFeatures == null) {
+            return new ArrayList<>();
+        }
+        return includedFeatures;
+    }
+
+    public void setIncludeFeatures(List<Feature> includedFeatures) {
+        this.includedFeatures = includedFeatures;
+    }
+
+    public List<Advice> getAdviceFileContent() {
+        return adviceFileContent;
+    }
+
+    public void setAdviceFileContent(List<Advice> adviceFileContent) {
+        this.adviceFileContent = adviceFileContent;
+    }
+
+    public RepositorySystem getRepositorySystem() {
+        return repositorySystem;
+    }
+
+    public void setRepositorySystem(RepositorySystem repositorySystem) {
+        this.repositorySystem = repositorySystem;
+    }
+
+    public ArtifactRepository getLocalRepository() {
+        return localRepository;
+    }
+
+    public void setLocalRepository(ArtifactRepository localRepository) {
+        this.localRepository = localRepository;
+    }
+
+    public List<ArtifactRepository> getRemoteRepositories() {
+        return remoteRepositories;
+    }
+
+    public void setRemoteRepositories(List<ArtifactRepository> remoteRepositories) {
+        this.remoteRepositories = remoteRepositories;
+    }
+
+    public MavenProject getProject() {
+        return project;
+    }
+
+    public void setProject(MavenProject project) {
+        this.project = project;
+    }
+
+    public MavenProjectHelper getProjectHelper() {
+        return projectHelper;
+    }
+
+    public void setProjectHelper(MavenProjectHelper projectHelper) {
+        this.projectHelper = projectHelper;
+    }
+
+    public void setLog(Log logger) {
+        this.log = logger;
+    }
+
+    public Log getLog() {
+        return this.log;
+    }
+
+    public File getPropertyFileInResourceDir() {
+        File propertyFile = new File(getProject().getBasedir() + "/src/main/resources/feature.properties");
```

Comment:
Refined the code using Path variables to construct the path.

Fix is in https://github.com/wso2/carbon-maven-plugins/pull/23/files#diff-c0f2ea4f80a32498ab34ffdac8b2f0fcL240